examples/abitest/module_1/rust/src/lib.rs

```diff
@@ -61,15 +61,15 @@ fn inner_main(in_handle: u64) -> Result<(), oak::OakStatus> {
 
     let out_channel = oak::WriteHandle { handle: out_handle };
     // Wait on 1+N read handles (starting with N=0).
-    let mut wait_handles = vec![in_channel];
+    let mut wait_handles = vec![in_channel.clone()];
```

Comment:
Do handles need to be cloned in order to be waited on now? I thought it would be nice if refs can be passed instead

Reply:
References work too, but in this case the other handles push onto this `Vec` need to be owned to keep them from being dropped, so I've chosen to clone this handle here for convenience.